src/argilla_sdk/settings/_resource.py

```diff
@@ -77,11 +77,11 @@ def __init__(
     #####################
 
     @property
-    def fields(self) -> List[FieldType]:
+    def fields(self) -> List[TextField]:
```

Comment:
We need to review fields, vectors, and metadata relationships, but what we have for now in `fields` are only `TextField`s